spec/system/page_title_spec.rb

```diff
@@ -0,0 +1,97 @@
+require 'rails_helper'
+
+def has_title(expected_title)
```

Comment:
We could use `RSpec::Matchers.define` so it reads 

```ruby
expect(root_path).to have_page_title('Home page')
```

Reply:
I'm not sure how this would work with the `yield`....

Reply:
If the assertion is inverted it might read better and there'd be no block or need to yield. I will try it to see if what I'm imagining makes sense.

Reply:
```ruby
require 'rspec/expectations'

RSpec::Matchers.define :have_page_title do |page_title|
  match do |path|
    visit path
    page.title.eql? "Child development training : #{page_title}"
  end
end

specify { expect('/').to have_page_title 'foo' }
```

Reply:
Or perhaps something like `have_title_suffix`

Reply:
> expected "/" to have title suffix "foo"

Reply:
I prefer the method I used - it reads better to me and has the benefit of working.  If I felt this was a matcher that could be used throughout the project, I'd consider it.  But this is a very specific set of tests...